packages/flutter_tools/lib/src/web/chrome.dart

```diff
@@ -79,11 +79,7 @@ class ChromeLauncher {
   /// `skipCheck` does not attempt to make a devtools connection before returning.
   Future<Chrome> launch(String url, { bool headless = false, bool skipCheck = false }) async {
     final String chromeExecutable = findChromeExecutable();
-    final Directory dataDir = fs.directory('.dart_tool')
-      .childDirectory('chrome_profile');
-    if (!dataDir.existsSync()) {
-      dataDir.createSync(recursive: true);
-    }
+    final Directory dataDir = fs.systemTempDirectory.createTempSync('flutter_tool_');
```

Comment:
I wonder if flutter_tools should maintain its own temp directory, perhaps inside the flutter installation. It would be easier to track all things that are generated/cached.

Reply:
One issue with the teardown (regardless of location) is that chrome takes a long time to actually exit, so we can't deleted the profile for several seconds after killing the chrome process (and even then it doesn't seem to reliably work).

If we kept our own cache we could check when spinning one up how many there are and delete inactive ones, to hopefully keep below a certain size?

One concern might be if this breaks, we'll fill up space that can't be so easily reclaimed